ee/tabby-db/migrations/0025_user-events.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE user_events (
+    id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    user_id INTEGER NOT NULL,
+    type TEXT NOT NULL,
```

Comment:
```suggestion
    kind TEXT NOT NULL,
```

Avoid aliasing and matching `EventKind`